iac-extensions/kubernetes/src/test/resources/checks/WhitespaceBracesCheckTest/helm/templates/whitespace-braces.yaml

```diff
@@ -23,3 +23,11 @@ spec:
 {{ print "cc: " | indent 6 }}{{ print .Values.containerName3}}
 {{ end}}
 
+{{/* Compliant */}}
+{{- /* Compliant */ -}}
```

Comment:
What about other examples with a dash and expression that is not a comment (maybe for documentation purposes only)? AFAIU, we will allow things like
```
{{-.Values }}
{{ .Values-}}
{{-.Values-}}
```
and raise only on one side for
```
{{.Values-}}
{{-.Values}}
```